packages/tsconfig.base.json

```diff
@@ -8,7 +8,6 @@
 
     "moduleResolution": "Bundler",
     "allowImportingTsExtensions": true,
-    "verbatimModuleSyntax": true,
```

Comment:
https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax

As far as I understand this, this option was not supposed to ever work and translate common JS files but we are using it like this right now in our `.cts` file. The docs suggest using a syntax like this instead:

```js
import foo = require('foo');

export = {
  foo
}
```

We use the `export =` syntax already but changing the `import` accordingly breaks with another TS issue:

```
│ src/compat/colors.cts(1,1): error TS1202: Import assignment cannot be used when targeting EC
│ MAScript modules. Consider using 'import * as ns from "mod"', 'import {a} from "mod"', 'impo
│ rt d from "mod"', or another module format instead.
```

I don't know how to fix this other than removing `verbatimModuleSyntax` unfortunately 😞  cc @tailwindlabs/engineering

Reply:
cc @RobinMalfait @thecrypticace what are your thoughts?